src/main/kotlin/racingcar/RacingCarApplication.kt

```diff
@@ -0,0 +1,18 @@
+package racingcar
+
+fun main() {
+    val inputView = InputView()
+    val resultView = ResultView()
+
+    println("자동차 대수는 몇 대인가요?")
+    val carNumber = inputView.input()
+
+    val carList = mutableListOf<Car>()
+    for (i in 1..carNumber) {
+        carList.add(Car())
+    }
```

Comment:
foreach가 더 가독성을 좋게 만들지 않을까요?

Reply:
@Flamme1004K `foreach`로 어떻게 추가할 수 있을까요 ? 🤔

Reply:
앗! foreach를 잘못 썻네요~ List 함수로도 충분히 가능한 부분이였습니다!

```suggestion
    val carList = List(carNumber) {
        Car()
    }
```